docs/getting-started/react-native-ios.md

```diff
@@ -151,12 +185,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     #if FB_SONARKIT_ENABLED
       let client = FlipperClient.shared()
       let layoutDescriptorMapper = SKDescriptorMapper(defaults: ())
-      FlipperKitLayoutComponentKitSupport.setUpWith(layoutDescriptorMapper)
```

Comment:
This should work if you add the pod ` pod 'FlipperKit/FlipperKitLayoutComponentKitSupport', versions['Flipper'], :configuration => 'Debug'`

Regarding